src/vivarium_public_health/population/data_transformations.py

```diff
@@ -124,21 +152,13 @@ def rescale_binned_proportions(pop_data, age_start, age_end):
         pop_data.loc[max_bin.index, "age_end"] = age_end
         pop_data.loc[padding_bin.index, "age_start"] = age_end
 
-    return pop_data
+    return pop_data[col_order].sort_values(_SORT_ORDER).reset_index(drop=True)
```

Comment:
Added some sort order guarantees to fix some busted tests that were relying on unenforced implementation details.

Reply:
Everything past here in this file is formatting